src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -281,23 +281,27 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
         case gjre: GoogleJsonResponseException if gjre.getDetails.getCode == StatusCodes.Conflict.intValue => None
       }
       _ <- creationOperationId match {
-        case Some(opId) => pollShellProjectCreation(opId) //poll until it's created
+        case Some(opId) => for {
+          _ <- pollShellProjectCreation(opId) //poll until it's created
+          enableStorageOpId <- googleServiceManagerDAO.enableService(projectName, "storage-api.googleapis.com")
+          _ <- pollServiceOperation(enableStorageOpId) //poll until it's enabled
+        } yield ()
```

Comment:
Nested `for` comprehensions, maybe keep this as an extracted method for readability?